source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOAdaptiveCard.mm

```diff
@@ -56,40 +56,115 @@ - (NSData *)inputs
     return _inputs;
 }
 
-+ (ACOAdaptiveCardParseResult *)fromJson:(NSString *)payload;
-{
++ (NSString *)correctInvalidJsonEscapes:(NSString *)payload {
+    NSError *regexError = nil;
+    NSRegularExpression *regex = [NSRegularExpression regularExpressionWithPattern:@"(\\\\+\\.)"
+                                                                           options:0
+                                                                             error:&regexError];
+    if (regexError != nil) {
+        NSLog(@"Regex Error: %@", regexError.localizedDescription);
```

Comment:
Do we need to be logging this? We should consider making this more "informational" sounding. It seems like hitting this line of code isn't necessarily an error.

Reply:
Technically this could be an error if there is malformed regex, and it would be caught with the next check under 'isValidJson' 

I can remove this log and do something more informal sounding like you suggest though

Reply:
Sorry, I misread the code, you are right. Let's keep the log 👍